azure-pipelines-release.yml

```diff
@@ -21,28 +21,44 @@ jobs:
         osx - python37:
           image.name: 'macos-12'
           python.version: '3.7'
+          ciwb.build: cp37-macosx_x86_64
+          ciwb.archs: x86_64
         osx - python38:
           image.name: 'macos-12'
           python.version: '3.8'
+          ciwb.build: cp38-macosx_x86_64
+          ciwb.archs: x86_64
         osx - python39:
           image.name: 'macos-12'
           python.version: '3.9'
+          ciwb.build: cp39-macosx_universal2
+          ciwb.archs: universal2
         osx - python310:
           image.name: 'macos-12'
           python.version: '3.10'
+          ciwb.build: cp310-macosx_universal2
+          ciwb.archs: universal2
```

Comment:
My understanding is that `universal2` packages both ARM and x86 builds into a single wheel, distributing both, making it potentially twice a big (in terms of MB) as individual builds. Wouldn't it make more sense to build individual wheels for both ARM and x86 chipsets? I'm thinking from a pure distribution standpoint here for Orange. The Orange distribution sizes have been a bit big for a while, and wouldn't this just exacerbate the problem?

Reply:
Also, why is `universal2` used only in Python versions 3.9 and 3.10? Why not 3.8 and 3.7 as well? Does `universal2` support only these versions of python?

Reply:
> My understanding is that universal2 packages both ARM and x86 builds into a single wheel, distributing both, making it potentially twice a big (in terms of MB) as individual builds. Wouldn't it make more sense to build individual wheels for both ARM and x86 chipsets? I'm thinking from a pure distribution standpoint here for Orange. The Orange distribution sizes have been a bit big for a while, and wouldn't this just exacerbate the problem?

400KB wont make a difference.

> Does universal2 support only these versions of python?

Yes (3.8 has some support but it is broken).